Makefile

```diff
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.0
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.14
```

Comment:
Why are we downgrading versions?

Reply:
The module and CI don't currently work with TF v0.15+. To support TF v0.15+ and TPG 4+ will be a longer/larger change, but if you prefer we can can wait on this commit and combine all the changes in a few days.

Reply:
In that case lets skip constraining the version in the module and just target this as a CI fix. Curious what happens with  TF v0.15+ that is blocking?

Reply:
A major addition with TF v0.15 is sensitive output values, so that change is my initial suspect for what is impacting the k8s auth during the CI.